packages/suite/src/components/suite/modals/AdvancedCoinSettings/components/BitcoinAmountUnit.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+import { H3, SelectBar, variables } from '@trezor/components';
+import { useBitcoinAmountUnit } from '@wallet-hooks/useBitcoinAmountUnit';
+import { Translation } from '@suite-components/Translation';
+import { PROTO } from 'packages/connect/lib';
+
+const StyledSelectBar = styled(SelectBar)`
+    margin-bottom: 28px;
+
+    ${variables.SCREEN_QUERY.MOBILE} {
+        margin-top: 10px;
+    }
+`;
+
+export const BitcoinAmountUnit = () => {
```

Comment:
@hynek-jina It's not in [the issue description](https://github.com/trezor/trezor-suite/issues/3004#issue-745551815), please add it there or tell us to delete it from the code

<img width="781" alt="image" src="https://user-images.githubusercontent.com/3729633/180986759-b424d257-7da2-4b28-bffd-d6df8e4b8a11.png">

Reply:
I've improved the issue specification.. (to previous version)
It should be here in bitcoin settings.. but not in general application settings.

Reply:
😢  I like it more in general settings, but of course that's up to you. 

Here in advanced coin settings it looks more local (one coin only), but it applies to testnet and regtest too, but not a big deal. And there is still a (quite hidden) way to change it by clicking on the amount on dashboard or account listing...

Reply:
I'm sorry for the confusion and changing the opinion.. But after discussion with you I would leave it in general settings.
@dahaca Told me that it will be visible only with activated one of the coins - `Bitcoin`, `Bitcoin testnet`, or `Bitcoin regtest`
- I will enhance the issue description by these two points.

Reply:
done